### DISCUSSION TITLE
url on phone?

### DISCUSSION THREAD

**Discussion by mg92865** [Budibase/budibase#14187] [Help] [answered]
I have been away 6 months.  Now I am able to log in as admin on laptop works.  Does not work correctly on Android, although it used to.  It now takes me to a small (reduced size) page that is not usable.  How to obtain normal size screen.

Thanks,

Mike

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mg92865,

Can you provide some screenshots of what you are seeing? When I try to login using my mobile this is what I see.

![451332543_877734021070446_5419714167229939819_n](https://github.com/user-attachments/assets/81db6af7-a22d-4199-85e3-9afeb00fb1fe)
